### ISSUE TITLE
Scope: which types of authoring tools do we include?

### ISSUE THREAD

**Issue by hidde** [w3c/wai-authoring-tools#4]
There are a lot of different authoring tools, as [Defining authoring tools · w3c/wai-authoring-tools Wiki](https://github.com/w3c/wai-authoring-tools/wiki/Defining-authoring-tools) shows.

We should decide which of these are in scope for the Authoring Tools List.

**Comment by hidde** [comment]
@yatil suggested in [EO call](https://www.w3.org/2019/08/02-eo-minutes.html):

> Self-hosted and hosted tools to publish content to individual websites (not social media)

**Comment by eoncins** [comment]
To me leaving social media out it may depend on how important is the content creation compared to the design and development.

Under Purpose>Page it states "Introduce people to authoring tool accessibility" to me people is "general public" not "experts", but then as target audience mention is only made to "experts": designers, developers, procurers, content editors, accessibility auditors, QA testers, project managers.

**Comment by yatil** [comment]
I think we must be careful to not have a too broad scope: Adding social media networks to the list might drown out the CMS vendors. I also think that most people won’t pick a social network by the accessibility of that network but considering if their audience is at that network. Accessibility aware organizations have worked around missing features in those cases. For example before alt text was available on Twitter, users often added the alt text in a tweet or a follow-up tweet.

**Comment by yatil** [comment]
As for the audience, I think seeing the list can have an effect on everyone, but it will be mostly useful to the specified audience groups.

**Comment by hidde** [comment]
@yatil: thanks, I really like “would be a product for which people would want to filter for accessibility features”, “no workarounds available” as criteria for whether to include in the scope, in addition to “self-hosted/hosted tools to publish web content”. 

For me, that would mean, these are included:

* CMS
* courseware
* website creators

These excluded:

* social media
* “Save as HTML”

That may leave these in a gray area:

* wikis
* online forums
* WYSIWYG editors
* templates

**Comment by yatil** [comment]
I think wikis and forum software would be good to have in there, WYSIWYG and templates might be too specific and “cluttering” the tool.

**Comment by hidde** [comment]
Sounds good to me, have added wikis and forum software to latest version.

**Comment by HelenBurge** [comment]
I think social media should be included as we are assuming the audience of this work will be organisations.  I am aware of a lot of users of social media using the sites to host their business pages as the main templates and structure are easy to use compared to a CMS.  I think adding in social media as an option is worthwhile as a different option from a more complex WordPress.

From my experience with the different authoring tools, most rely on mouse only interaction to build a web page.  Social media provides a solution that can be set up using no mouse as the structure and layout is preset.

**Comment by yatil** [comment]
If we want to have Social media highlighted in this way, we should have a second resource for them. I see this resource more for projects that are operated by the people using the tool instead of being operated by a 3rd party.

Basically: Can you make substantial changes to the software or its output yourself, then it is a candidate for this specific list.

**Comment by HelenBurge** [comment]
You create the page title, content, images (weirdly no alt tags for cover photos), change the template, add items like buttons with embedded links. So the branding of say "Facebook" will never disappear, the main content is decided by you like a WordPress created site using templates.

**Comment by yatil** [comment]
Yes, but I cannot change to a more accessible Twitter template or Facebook template. I think there is a difference and putting all into one tool will make it harder to create and maintain filters. I’m all for adding social media at some point in some form, but I think we need to scope this narrowly first to be effective.

**Comment by hidde** [comment]
I agree with @HelenBurge that it would be useful to provide information to people who use social media, as it is a lot and becoming more and more common to be used by small businesses instead of a “real” website.

But I think, like @yatil, that this category is so different from the other tools that it warrants its own resource and would like us to leave it out of scope for he first iteration of the project.

**Comment by nitedog** [comment]
I'm thinking of social media software that you can install locally within an organization, much like a wiki or CMS software product. I think these types of tools should be included, not necessarily the instances of such tools (we also speak about "wiki" not of "wikipedia", so I agree that speaking of "Facebook" or "Twitter" is not the point but maybe the software behind).

**Comment by slhenry** [comment]
It depends on the purpose of the list, e.g.:
* to help people pick an authoring tool to use or purchase
* to showcase accessibility status of authoring tools

**Comment by hidde** [comment]
Today we discussed this issue in EOWG.

We talked about @slhenry's comment regarding purpose of the list, and agreed it is both for helping people pick tools and for showcasing accessibility status/features. We'll focus on the picking tools aspect first.

Multiple distinct uses of social media emerged: 

- a (likely small) company uses Facebook to create their online presence (the social medium becomes a “website creator”)
- companies use internal social networks (like Yammer by Microsoft or Workplace by Facebook)
- companies use social media in a traditional sense (to communicate/engage with clients)

We also talked about broadening and/or redefining the categories, @HelenBurge volunteered to look at this and map possible categories with descriptions to examples.

**Comment by HelenBurge** [comment]
Added a new task with high level list (no actual brands used): https://github.com/w3c/wai-authoring-tools/issues/62
